packages/trpc/server/routers/viewer/webhook/delete.handler.ts

```diff
@@ -13,14 +13,20 @@ type DeleteOptions = {
 export const deleteHandler = async ({ ctx, input }: DeleteOptions) => {
   const { id } = input;
 
-  const andCondition: Partial<{ id: string; eventTypeId: number; teamId: number; userId: number }>[] = [
-    { id: id },
-  ];
+  const andCondition: Partial<{
+    id: string;
+    eventTypeId: number;
+    teamId: number;
+    userId: number;
+    platform: boolean;
+  }>[] = [{ id: id }];
 
   if (input.eventTypeId) {
     andCondition.push({ eventTypeId: input.eventTypeId });
   } else if (input.teamId) {
     andCondition.push({ teamId: input.teamId });
+  } else if (ctx.user.role == "ADMIN") {
+    andCondition.push({ platform: true });
```

Comment:
with this, the ADMIN won't be able to delete any of their other event types anymore

Reply:
i see 🤔